stdlib/format.mli

```diff
@@ -30,14 +30,22 @@
    - {!std_formatter} outputs to {{!Stdlib.stdout}stdout}
    - {!err_formatter} outputs to {{!Stdlib.stderr}stderr}
 
-   Most functions in the {!Format} module come in two variants:
-   a short version that operates on {!std_formatter} and the
-   generic version prefixed by [pp_] that takes a formatter
-   as its first argument.
+   Most functions in the {!Format} module come in two variants: a short version
+   that operates on the current domain's {!std_formatter} and the generic
```

Comment:
> I also wonder if it would be a good idea to clarify that the get_std_formatter call is internally performed after the last argument is received.

Good point. I've added

```
For the version that operates on the current domain's standard formatter, the 
call to {!get_std_formatter} is delayed until the last argument is received.
```